src/singleton.activation.js

```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line no-unused-vars
+var wwtActivationSingleton = ( function () {
+	var $parserOutput = $( '.mw-parser-output' ),
+		$parserOutputStored = $parserOutput.clone(),
+		interfaceLang = $( 'html' ).attr( 'lang' ),
+		/**
+		 * Initialize the activation button to toggle WhoWroteThat system.
+		 * Only attaches the button for pages in the main namespace, and only
+		 * in view mode.
+		 *
+		 * @param  {[type]} allTranslations An object representing all available
+		 *  translations, keyed by language code.
+		 * @param  {[type]} onClickFunction The function that is triggered when
+		 *  the activation button is clicked.
+		 */
+		initialize = function ( allTranslations, onClickFunction ) {
+			var link;
+
+			// Load all messages
+			mw.messages.set(
+				$.extend( {},
+					// Manually create fallback on English
+					allTranslations.en,
+					allTranslations[ interfaceLang ]
+				)
+			);
+
+			// Add a portlet link to 'tools'
+			link = mw.util.addPortletLink(
+				'p-tb',
+				'#',
+				mw.msg( 'ext-whowrotethat-activation-link' ),
+				't-whowrotethat',
+				mw.msg( 'ext-whowrotethat-activation-link-tooltip' ),
+				'',
+				'#t-print'
```

Comment:
I don't think it really matters, but in English Wikipedia the t-print link has been moved out of the toolbox and into a 'Print/export' portlet. So our link is added at the end (with whatever other gadgets do the same).